audio/encoding.py

```diff
@@ -20,6 +22,7 @@ def __init__(
         fixed_bitrate=None,
         force_num_channels=None,
         select_channels=None,
+        recover_duration=None,
```

Comment:
maybe add at the end of the argument list so that old setups that give positional args don't break